android-o/AndroidPlayLocation/LocUpdFgService/LocUpdFgService/LocationUpdatesService.cs

```diff
@@ -109,12 +114,20 @@ public override void OnCreate()
 			LocationCallback = new LocationCallbackImpl { Service = this };
 
 			CreateLocationRequest();
+            GetLastLocation();
 
 			HandlerThread handlerThread = new HandlerThread(Tag);
 			handlerThread.Start();
 			ServiceHandler = new Handler(handlerThread.Looper);
-			NotificationManager = (NotificationManager) GetSystemService(NotificationService);
-		}
+			NotificationManager = (NotificationManager) GetSystemService(NotificationService);
+
+		    if (Build.VERSION.SdkInt >= Build.VERSION_CODES.O)
+		    {
+		        var name = (String)GetString(Resource.String.app_name);

```

Comment:
This cast is throwing on startup, we'll need to update it.

Reply:
Done.